textile/fabric_printing/doctype/print_order/print_order.py

```diff
@@ -860,9 +860,9 @@ def _create_design_items_and_boms(self, publish_progress=True, ignore_version=Tr
 				item_doc = self.make_design_item(d)
 				item_doc.flags.ignore_version = ignore_version
 				item_doc.flags.ignore_feed = ignore_feed
-				item_doc.flags.ignore_permissions = True
+				item_doc.flags.ignore_permissions = ignore_permissions
```

Comment:
This should always be True. Check permissions separately before creating document